### ISSUE TITLE
Open source license

### ISSUE THREAD

**Issue by pradee1p** [IzakMarais/reporter#259]
Hello, This reporter plugin is completely open source? Or you have used any of the Grafanna enterprise features? (For example: PDF generator). Also is the PDFLATEX opensource? Kindly clarify.

**Comment by IzakMarais** [comment]
Yes it is completely open source. It did not use any enterprise grafana features. I'm not sure about pdflatex, n but I suspect it is. 


________________________________
